ncm-metaconfig/src/main/metaconfig/conntrackd/pan/schema.pan

```diff
@@ -0,0 +1,117 @@
+declaration template metaconfig/conntrackd/schema;
+
+include 'pan/types';
+
+@documentation{
+    Conntrackd config
+}
+
+@documentation{
+    There are 3 main synchronization modes or protocols: NOTRACK, ALARM and FTFW.
+}
+type conntrackd_service_sync_mode = {
+    'type' : choice('FTFW', 'ALARM', 'NOTRACK,') = 'FTFW'
+    'DisableExternalCache' ? boolean
+    'CommitTimeout' ? long
+    'PurgeTimeout' ? long
+};
+
+@documentation{
+    There are 3 transport protocols: TCP, Multicast and UDP.
+}
+type conntrackd_service_sync_transport = {
+    'type' : choice('TCP', 'Multicast', 'UDP')
+    'IPv4_address' ? type_ipv4
+    'IPv6_address' ? type_ipv6
+    'IPv4_Destination_Address' ? type_ipv4
+    'IPv6_Destionation_Address' ? type_ipv6
+    'Port' ? long
+    'Interface'? string
+    'IPv4_interface' ? string
+    'SndSocketBuffer' ? long
+    'RcvSocketBuffer' ? long
+    'Checksum' ? boolean
+};
+
+@documentation{
+    This top-level section defines how conntrackd should handle synchronization with other cluster nodes.
+}
+type conntrackd_service_sync = {
+    'mode' : conntrackd_service_sync_mode
+    'transport' : conntrackd_service_sync_transport[]
+};
+
+@documentation{
+    Unix socket configuration.
+    This socket is used by conntrackd to listen to external commands like `conntrackd -k' or `conntrackd -n'.
+}
+type conntrackd_service_general_unix = {
+    'Path' : string = '/var/run/conntrackd.ctl'
+    'Backlog' : long = 20
```

Comment:
The man page I'm looking at suggests this is a deprecated option, should we be including it?

Reply:
ah, it's not in the manpage of the conntrack-tools-1.4.4-10.el8 package on rhel8, but indeed on fedora i see the same

Reply:
Fair enough, can you put some bounds on the number then?

Reply:
I haven't found any meaningful documentation for this option, other than the deprecation commit which states:
> This configuration option doesn't add any value to users.

So I suppose it can remain as it is.